Tests/Common/LnsVersionRequest.cs

```diff
@@ -8,25 +8,25 @@ namespace LoRaWan.Tests.Simulation.Models
     public sealed record LnsVersionRequest
```

Comment:
This is serialized-only in one place so doesn't seem to warrant a whole separate record to maintain:

https://github.com/Azure/iotedge-lorawan-starterkit/blob/4490b3287bdc166a1d6b728d504abfb0b1377a6f/Tests/Common/SimulatedBasicsStation.cs#L55

What's more, in two other cases we just serialize an anonymous version; here:

https://github.com/Azure/iotedge-lorawan-starterkit/blob/4490b3287bdc166a1d6b728d504abfb0b1377a6f/Tests/Common/SimulatedBasicsStation.cs#L44

and here:

https://github.com/Azure/iotedge-lorawan-starterkit/blob/4490b3287bdc166a1d6b728d504abfb0b1377a6f/Tests/Common/SimulatedBasicsStation.cs#L81-L102

So just thinking, why not remove this entirely and follow the same route for consistency?